spaghetto/spago.yaml

```diff
@@ -45,11 +45,12 @@ package:
     - transformers
     - tuples
     - unsafe-coerce
+    - versions
```

Comment:
I think this can be removed. Originally, I thought I would need it because `psa` would output things to `stderr` or `stdout` depending on the `purs` version. Since spago only supports `0.15.4` or higher, that question is irrelevant now.